server/src/main/resources/application.example.yml

```diff
@@ -36,32 +36,32 @@ seata:
     type: file
     nacos:
       server-addr: 127.0.0.1:8848
-      namespace:
+      namespace: ""
       group: SEATA_GROUP
-      username:
-      password:
-      context-path:
+      username: ""
+      password: ""
+      context-path: ""
       ##if use MSE Nacos with auth, mutex with username/password attribute
       #access-key: ""
       #secret-key: ""
       data-id: seataServer.properties
     consul:
       server-addr: 127.0.0.1:8500
-      acl-token:
+      acl-token: ""
       key: seata.properties
     apollo:
       appId: seata-server
       apollo-meta: http://192.168.1.204:8801
       apollo-config-service: http://192.168.1.204:8080
       namespace: application
-      apollo-access-key-secret:
+      apollo-access-key-secret: ""
       cluster: seata
     zk:
       server-addr: 127.0.0.1:2181
       session-timeout: 6000
       connect-timeout: 2000
-      username:
-      password:
+      username: ""
```

Comment:
What is the yml specification for strings and why quotes are not required for String configurations that now have non-null values.

Reply:
![image](https://user-images.githubusercontent.com/24457941/205466633-0f2ceff4-7bbe-4394-adbe-b7f4c877b56e.png)

Because we have assigned the empty string value in the properties class and Assigning a default value is also a best practice to avoid null values.

Reply:
I will review the properties class again and unify them.